Geometry/HGCalGeometry/test/HGCalGeomLocatorTester.cc

```diff
@@ -75,5 +81,36 @@ void HGCalGeomLocaterTester::doTest(const HGCalGeometry* geom, DetId::Detector d
             << " are bad\n\n\n\n";
 }
 
+void HGCalGeomLocaterTester::doTestScintillator(const HGCalGeometry* geom, DetId::Detector det) {
+  const std::vector<DetId>& ids = geom->getValidDetIds();
+  std::cout << "doTest:: " << ids.size() << " valid ids for " << geom->cellElement() << std::endl;
+  const double tol = 0.5;
+  const unsigned int step = 10;
+  int all(0), good(0), bad(0);
+  for (unsigned int k = 0; k < ids.size(); k += step) {
+    ++all;
+    HGCScintillatorDetId id(ids[k]);
+    if ((id.iradiusAbs() != 7) && (id.iradiusAbs() != 41)) {
```

Comment:
why are 7 and 41 special?

Reply:
Currently there are issues - which I need to investigate. For 41 I need to store the largest R - for 7 most likely the issue was the readjustment of some R-values. I found out those issues when I introduced this new method. I shall work on them later